db/migrate/20240111111959_add_user_ref_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToComments < ActiveRecord::Migration[7.1]
+  def change
+    add_reference :comments, :author, foreign_key: { to_table: 'users' }
+  end
```

Comment:
- [ ] As per the ERD diagram, The foreign key between comments and users tables should be `user_id`, not `author_id` as you did here. Please fix that.

![image](https://github.com/yusufyusie/blog_app/assets/68796232/1e1309d3-cdb4-4a63-b447-e2f1cc66e666)

```suggestion
  def change
    add_reference :comments, :user, null: false, foreign_key: true
  end
```